src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -116,41 +116,94 @@ public async Task it_can_load_assembly_references_using_r_directive_separate_sub
         [InlineData(Language.FSharp, false)]
         [InlineData(Language.CSharp, true)]
         //[InlineData(Language.FSharp, true, Skip = "FSharp, relative is not relative to cwd.")]
-        public async Task it_can_load_assembly_references_using_r_directive_with_relative_path(Language language, bool changeWorkingDirectory)
+        public async Task it_can_load_assembly_references_using_r_directive_with_relative_path(Language language, bool changeCurrentDirectoryInUserCode)
         {
-            var workingDirectory = Directory.GetCurrentDirectory();
-            DisposeAfterTest(() => Directory.SetCurrentDirectory(workingDirectory));
+            var currentDirectory = Directory.GetCurrentDirectory();
+            DisposeAfterTest(() => Directory.SetCurrentDirectory(currentDirectory));
 
             var kernel = CreateKernel(language);
 
-            if (changeWorkingDirectory)
+            if (changeCurrentDirectoryInUserCode)
             {
+                //Even when a user changes the current directory, loading from a relative path is not affected.
                 await kernel.SendAsync(new SubmitCode("System.IO.Directory.SetCurrentDirectory(\"..\")"));
             }
 
-            var fullName = new FileInfo(typeof(JsonConvert).Assembly.Location).FullName;
+            var dllName = new FileInfo(typeof(JsonConvert).Assembly.Location).Name;             
```

Comment:
whitespace

Reply:
sorry, probably I'm whitespace-blind.

Reply:
fixed in https://github.com/dotnet/interactive/pull/1176/commits/0dd55f30e66ea43e619e2fa75ba4cccdbbae6f6e